make/00_mod.mk

```diff
@@ -22,6 +22,18 @@ oci_preflight_image_name := quay.io/jetstack/venafi-agent
 oci_preflight_image_tag := $(VERSION)
 oci_preflight_image_name_development := jetstack.local/venafi-agent
 
+# Annotations are the standardised set of annotations we set on every component we publish
+oci_preflight_build_args := \
+	--image-annotation="org.opencontainers.image.vendor"="CyberArk Software Ltd." \
+	--image-annotation="org.opencontainers.image.licenses"="EULA - https://www.cyberark.com/contract-terms/" \
+	--image-annotation="org.opencontainers.image.authors"="support@venafi.cloud" \
+	--image-annotation="org.opencontainers.image.title"="Venafi Kubernetes Agent" \
+	--image-annotation="org.opencontainers.image.description"="Gathers machine identity data from Kubernetes clusters." \
+	--image-annotation="org.opencontainers.image.url"="https://www.cyberark.com/products/certificate-manager-for-kubernetes/" \
+	--image-annotation="org.opencontainers.image.documentation"="https://docs.venafi.cloud/vaas/k8s-components/c-tlspk-agent-overview/" \
+	--image-annotation="org.opencontainers.image.version"="$(VERSION)" \
+	--image-annotation="org.opencontainers.image.revision"="$(GITCOMMIT)"
```

Comment:
question: Where do these values come from? How can I evaluate whether these are what the product team want?

Reply:
This document describes all the annotations we want: https://venafi.atlassian.net/wiki/spaces/TPFK/pages/3094839338/OCI+labels+and+annotations